packages/mui-joy/src/Autocomplete/Autocomplete.tsx

```diff
@@ -401,6 +401,20 @@ const Autocomplete = React.forwardRef(function Autocomplete(
 
   const classes = useUtilityClasses(ownerState);
 
+  const modifiers = React.useMemo(
+    () => [
+      {
+        name: 'offset',
+        options: {
+          offset: [0, 4],
+        },
+      },
+      ...((props.slotProps?.listbox as Omit<PopperUnstyledOwnProps, 'slots' | 'slotProps' | 'open'>)
+        ?.modifiers || []),
+    ],
+    [props.slotProps?.listbox],
```

Comment:
I think it should be this
```suggestion
    [props.slotProps?.listbox?.modifiers],
```

Reply:
![Screenshot 2023-04-04 at 17 09 58](https://user-images.githubusercontent.com/32841130/229852611-800a1701-33b1-4ff6-84aa-e1d2ca788e7b.png)

I get this error. Do you know why?